vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -492,17 +489,6 @@ class OnboardingViewModel @AssistedInject constructor(
 
     private fun handleInitWith(action: OnboardingAction.InitWith) {
         loginConfig = action.loginConfig
-        // If there is a pending email validation continue on this step
-        try {
```

Comment:
Removing the attempt to fast forward the onboarding state to the _wait for email verification stage_ when the app has been killed and relaunched is the fix for #6860 

The `OnboardingViewModel` and `OnboardingViewState` already support and are restored by the system restoration process. If the process has been destroyed we need to accumulate the previous view states again otherwise we end up with invalid state.